src/libs/ReportUtils.ts

```diff
@@ -1857,7 +1868,12 @@ function findSelfDMReportID(): string | undefined {
         return;
     }
 
+    if (selfDMReportID) {
+        return selfDMReportID;
+    }
+
     const selfDMReport = Object.values(allReports).find((report) => isSelfDM(report) && !isThread(report));
+
     return selfDMReport?.reportID;
```

Comment:
Would be great to cover this logic in Unit tests if there are none yet, maybe lets look into that in a follow up